Wikipedia/Code/WMFExploreSectionControllerCache.m

```diff
@@ -52,9 +47,7 @@ - (instancetype)initWithDataStore:(MWKDataStore*)dataStore {
     if (self) {
         self.dataStore                              = dataStore;
         self.sectionControllersBySection            = [[NSMutableDictionary alloc] init];
-        self.reverseLookup                          =
-            [NSMapTable mapTableWithKeyOptions:NSMapTableWeakMemory | NSMapTableObjectPointerPersonality
-                                  valueOptions:NSMapTableWeakMemory];
+        self.reverseLookup                          = [[NSMutableDictionary alloc] init];
```

Comment:
Do we no longer need this to be weak? Is it expected that the new behavior will take care of cleaning up all the objects in the reverse lookup?

Also, is it ok that the section controllers don't conform to NSCopying - will the dictionary be able to find them based on pointer equality?